docs/reference/docs/bulk.asciidoc

```diff
@@ -66,17 +66,97 @@ example of a correct sequence of bulk commands:
+POST _bulk
 { "index" : { "_index" : "test", "_type" : "type1", "_id" : "1" } }
 { "field1" : "value1" }
 { "delete" : { "_index" : "test", "_type" : "type1", "_id" : "2" } }
 { "create" : { "_index" : "test", "_type" : "type1", "_id" : "3" } }
 { "field1" : "value3" }
-{ "update" : {"_id" : "1", "_type" : "type1", "_index" : "index1"} }
+{ "update" : {"_id" : "4", "_type" : "type1", "_index" : "index1"} }
 { "doc" : {"field2" : "value2"} }
 --------------------------------------------------
+// CONSOLE
+
+The result of this bulk operation is:
 
-In the above example `doc` for the `update` action is a partial
-document, that will be merged with the already stored document.
+[source,js]
+--------------------------------------------------
+{
+   "took": ...,
+   "errors": true, <1>
+   "items": [
+      {
+         "index": { <2>
+            "_index": "test",
+            "_type": "type1",
+            "_id": "1",
+            "_version": 1,
+            "result": "created",
+            "_shards": {
+               "total": 2,
+               "successful": 1,
+               "failed": 0
+            },
+            "created": true,
+            "status": 201
+         }
+      },
+      {
+         "delete": { <3>
+            "found": false,
+            "_index": "test",
+            "_type": "type1",
+            "_id": "2",
+            "_version": 1,
+            "result": "not_found",
+            "_shards": {
+               "total": 2,
+               "successful": 1,
+               "failed": 0
+            },
+            "status": 404
+         }
+      },
+      {
+         "create": { <4>
+            "_index": "test",
+            "_type": "type1",
+            "_id": "3",
+            "_version": 1,
+            "result": "created",
+            "_shards": {
+               "total": 2,
+               "successful": 1,
+               "failed": 0
+            },
+            "created": true,
+            "status": 201
+         }
+      },
+      {
+         "update": { <5>
+            "_index": "index1",
+            "_type": "type1",
+            "_id": "4",
+            "status": 404,
+            "error": {
+               "type": "document_missing_exception",
+               "reason": "[type1][4]: document missing",
+               "index_uuid": ...,
+               "shard": "2",
+               "index": "index1"
+            }
+         }
+      }
+   ]
+}
+--------------------------------------------------
+// TESTRESPONSE[s/"took": .../"took": $body.took/ s/"index_uuid": .../"index_uuid": $body.items.3.update.error.index_uuid/]
+<1> Some operation(s) in the _bulk request failed
+<2> The first operation succeeded with 201 status
+<3> The delete operation failed because the targeted document does not exist
+<4> The create operation succeeded with 201 status
+<5> The update operation failed because the targeted document is missing (create and update the same document in a single _bulk request does not work)
```

Comment:
This is incorrect - it didn't fail for the reason stated but because you're trying to update a document that hasn't been created.